sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryLevelExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package io.sentry.kotlin.multiplatform.extensions
+
+import io.sentry.kotlin.multiplatform.AndroidSentryLevel
+import io.sentry.kotlin.multiplatform.SentryLevel
+
+fun SentryLevel.toAndroidSentryLevel(): AndroidSentryLevel {
+    when (this) {
+        SentryLevel.DEBUG -> return AndroidSentryLevel.DEBUG
+        SentryLevel.INFO -> return AndroidSentryLevel.INFO
+        SentryLevel.WARNING -> return AndroidSentryLevel.WARNING
+        SentryLevel.ERROR -> return AndroidSentryLevel.ERROR
+        SentryLevel.FATAL -> return AndroidSentryLevel.FATAL
+    }
+    throw IllegalArgumentException("Sentry Level does not exist")
```

Comment:
Never do that, we could cause crashes on people's apps.
We can have sensible defaults or return null.